packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -744,6 +745,18 @@ export class Markets {
       marketCount: marketData.length,
     };
 
+    const orderFilledLogsByMarket = await getOrderFilledLogsByMarket(db, marketData);
+    marketData = marketData.map(market => {
+      const m = orderFilledLogsByMarket[market.market];
+      const numberOfTrades = m?.length || 0;
+      const feesGenerated = (new BigNumber(Number(market.feePercent))).times(numberOfTrades);
+
+      return {
+        ...market,
+        feesGenerated: feesGenerated.toNumber(),
+      }
+    });
```

Comment:
We can make this simpler and JUST sort by the number of trades :-D

```suggestion

      return {
        ...market,
        numberOfTrades,
      }
    });
```

Also, can you move this code into the MarketDB reducer? There's boiler plate in there which recalculates stuff on the MarketDB on a trade. That would result in the original query having a `numberOfTrades` property just like it has `lastTradeTimestamp` now -- and then the getter doesn't need to fetch all this data.

Reply:
👍 can do!